src/main/java/recipeio/CommandValidator.java

```diff
@@ -58,6 +58,24 @@ public static boolean isWord(String input) {
         return true;
     }
 
+    public static boolean isMealCat(String input) {
+        String coreInput = input.trim().toLowerCase();
+        boolean isCategory;
+        switch (coreInput) {
+        case Constants.MEAL_CAT_GENERAL:
+        case Constants.MEAL_CAT_DINNER:
+        case Constants.MEAL_CAT_LUNCH:
+        case Constants.MEAL_CAT_BREAKFAST:
+        case Constants.MEAL_CAT_APPETIZER:
+        case Constants.MEAL_CAT_DESSERT:
+            isCategory = true;
+            break;
+        default:
+            isCategory = false;
+        }
+        return isCategory;
+    }
+
```

Comment:
Could you add some feedback to the user saying that his input is invalid, and show the acceptable categories?